filter_info.go

```diff
@@ -42,39 +37,48 @@ const (
 
 )
 
-var ErrFieldExist = errors.New("Field is already registered")
+type Context struct {
+	Id               fieldId
+	IdToFieldNameMap map[fieldId]string
+	FieldNameToIdMap map[string]fieldId
+	IdToFieldTypeMap map[fieldId]ftenum
+}
 
-var id fieldId
+var ErrFieldExist = errors.New("gofilter: Field is already registered")

```

Comment:
Standard packages use lowercase characters like in [encoding\/hex](https://github.com/golang/go/blob/13529cc5f443ef4e242da3716daa0032aa8d34f2/src/encoding/hex/hex.go#L57)